twitter.py

```diff
@@ -21,16 +21,22 @@
 
 f = open(filepath, "a+")
 regex = r"(^|[^@\w])@(\w{1,30})"
-newtweetcounter, oldtweetcounter = 0, 0
+newtweetcounter, oldtweetcounter, ignorecounter = 0, 0, 0
 
-for tweet in tweepy.Cursor(api.search,q="concours RT",
-                           lang="fr",result_type='popular', tweet_mode='extended').items(int(os.getenv('SEARCH_ITEM_NUMBER'))):
+for tweet in tweepy.Cursor(api.search, q="concours RT",
+                           lang="fr", result_type='popular', count=int(os.getenv('SEARCH_ITEM_NUMBER')),
+                           tweet_mode='extended').items(int(os.getenv('SEARCH_ITEM_NUMBER'))):
 
     if int(tweet.id_str) in listTweetID:
-        if os.getenv('DEBUG') == 1:
+        if int(os.getenv('DEBUG')) == 1:
             print("tweet already processed : ",int(tweet.id_str))
         oldtweetcounter += 1
         continue
+    elif (tweet.in_reply_to_status_id is not None) or hasattr(tweet, 'retweeted_status'):
+        if int(os.getenv('DEBUG')) == 1:
+            print("tweet is a reply/RT : ",int(tweet.id_str))
+        ignorecounter += 1
```

Comment:
I think you should do this before the "if" line 36 because with debug == 0 you will not have this information in the display bellow (line 74)

Reply:
Only the print is displayed when debug mode is active. The ignore counter still increment whether debug mode is enabled or not. However, this can be moved for readability if you wish.